src/ProcessManager/WebServerReadinessProbeTrait.php

```diff
@@ -38,21 +40,41 @@ private function checkPortAvailable(string $hostname, int $port, bool $throw = t
 
     public function waitUntilReady(Process $process, string $url, bool $ignoreErrors = false): void
     {
-        $context = stream_context_create(['http' => [
-            'ignore_errors' => $ignoreErrors,
-            'protocol_version' => '1.1',
-            'header' => ['Connection: close'],
-            'timeout' => 5,
-        ]]);
-
-        while (Process::STATUS_STARTED !== ($status = $process->getStatus()) || false === @file_get_contents($url, false, $context)) {
-            if (Process::STATUS_TERMINATED === $status) {
-                throw new \RuntimeException($process->getErrorOutput(), $process->getExitCode());
+        $timeout = 5;
```

Comment:
Maybe add `$timeout` as a parameter of the method?

Reply:
fixed, thanks